packages/hint-typescript-config/src/target.ts

```diff
@@ -227,10 +229,48 @@ export default class TypeScriptConfigTarget implements IHint {
             return maxVersion;
         };
 
+        const browserslistConfigExists = (): boolean => {
+            let browserslistFolder: string | null;
+
+            try {
+                browserslistFolder = findPackageRoot(__dirname, '.browserslistrc');
+            } catch (e) {
+                browserslistFolder = null;
+            }
+
+            if (browserslistFolder) {
+                return true;
+            }
+
+            let packageJsonFolder: string | null;
+
+            try {
+                packageJsonFolder = findPackageRoot(__dirname, 'package.json');
+            } catch (e) {
+                packageJsonFolder = null;
+            }
+
+            if (packageJsonFolder) {
+                try {
+                    const packageJson = require(path.join(packageJsonFolder, 'package.json'));
+
+                    return !!packageJson.browserslist;
+                } catch (e) {
+                    console.log(e);
```

Comment:
Leftover? If not maybe use `debug(e)` and add some more info?

Reply:
totally leftover.